<?xml version="1.0" ?>
<sdf version="1.5">
  <world name= "person_walking">

    <actor name="person_walking">
      <static>false</static>
      <skin>
        <filename>walk.dae</filename>
        <scale>1.0</scale>
      </skin>

      <pose>0 0 0 0 0 0</pose>

      <animation name="walking">
          <filename>walk.dae</filename>
          <scale>1</scale>
          <interpolate_x>true</interpolate_x>
      </animation>

      <script>
          <loop>false</loop>
          <delay_start>0</delay_start>
          <auto_start>true</auto_start>
      </script>

      <link name="link">
        <inertial>
          <pose>0 0 0.95 0 0 0</pose>
          <mass>80.0</mass>
          <inertia>
            <ixx>27.82</ixx>
            <ixy>0</ixy>
            <ixz>0</ixz>
            <iyy>24.88</iyy>
            <iyz>0</iyz>
            <izz>4.57</izz>
          </inertia>
        </inertial>

        <collision name="bottom">
          <pose>0 0 0.01 0 0 0</pose>
          <geometry>
            <box>
              <size>0.35 0.75 0.02</size>
            </box>
          </geometry>
        </collision>

        <collision name="person">
          <pose>0 0 -0.02 0 0 0</pose>
          <geometry>
            <mesh>
              <uri>model://person_walking/meshes/walking.dae</uri>
            </mesh>
          </geometry>
        </collision>

        <visual name="visual">
          <pose>0 0 -0.02 0 0 0</pose>
          <geometry>
            <mesh>
              <uri>model://person_walking/meshes/walking.dae</uri>
            </mesh>
          </geometry>
        </visual>
      </link>
      
    </actor>
  </model>
</sdf>
